src/components/image-tooltip/ImageTooltip.tsx

```diff
@@ -31,8 +31,7 @@ const ImageTooltip = ({ children, className, content, image, title, ...otherTool
         </ImageTooltipContent>
     );
 
-    const imageTooltipClasses = classNames('bdl-ImageTooltip', {
-        className,
+    const imageTooltipClasses = classNames('bdl-ImageTooltip', className, {
```

Comment:
I think we should either make className default to an empty string or modify the object passed to classNames to include something like this:
```
{
    [className]: !!className,
}

Reply:
I think `classnames()` ignores `undefined` values (so we shouldn't need that check IMO). Wiola's change makes sense to me though we might want to see how other BUIE components are handling it.

Reply:
I've checked few components in BUIE and they mostly share my approach. 
Furthermore when I tried to modify object passed to classNames, I've got an error from TS:
![Screenshot 2021-12-23 at 10 38 39](https://user-images.githubusercontent.com/29651042/147232859-e86a0851-6a90-4d68-b803-880defe3f15f.png)

Reply:
Makes sense!! Thank you for checking